pkg/volume/cephfs/cephfs.go

```diff
@@ -242,7 +243,7 @@ func (cephfsVolume *cephfsMounter) SetUpAt(dir string, fsGroup *int64) error {
 		glog.V(4).Info("CephFS fuse mount.")
 		err = cephfsVolume.execFuseMount(dir)
 		// cleanup no matter if fuse mount fail.
-		keyringPath := cephfsVolume.GetKeyringPath()
+		keyringPath := path.Join(cephfsVolume.getConfigPath(), cephfsVolume.getKeyringFileName())
```

Comment:
use `filepath.Join` instead of `path.Join`

Reply:
Will change it soon.